crates/runtime/src/accelerated_table/refresh_task.rs

```diff
@@ -506,8 +515,23 @@ impl RefreshTask {
     }
 
     fn refresh_df_context(&self, federated_provider: Arc<dyn TableProvider>) -> SessionContext {
-        let ctx =
-            SessionContext::new_with_config_rt(get_df_default_config(), default_runtime_env());
+        let ctx = if self.disable_refresh_federation {
```

Comment:
I'm not sure I understand why this needs to be a configuration parameter. Why not just always use the federated query planner?